src/library/dao/cuedao.cpp

```diff
@@ -147,7 +154,7 @@ bool CueDAO::saveCue(Cue* cue) {
         query.bindValue(":length", cue->getLength());
         query.bindValue(":hotcue", cue->getHotCue());
         query.bindValue(":label", cue->getLabel());
-        query.bindValue(":color", cue->getColor()->m_iId);
+        query.bindValue(":color", static_cast<mixxx::RgbColor::code_t>(cue->getColor()));
```

Comment:
Is this cast actually needed? RgbColor::toQVariant() would be the correct way for binding database values.